network_files/config/genesis.json

```diff
@@ -0,0 +1,3494 @@
+                  "max_change_rate": "0.010000000000000000"
+                },
+                "min_self_delegation": "1",
+                "delegator_address": "zeta1hk05v9len8u0c2xrwxgfknvcskpd4vncm7ehch",
+                "validator_address": "zetavaloper1hk05v9len8u0c2xrwxgfknvcskpd4vncl7pt4p",
+                "pubkey": {
+                  "@type": "/cosmos.crypto.ed25519.PubKey",
+                  "key": "C1qu7Xgpn6/m+CGiqz5C8FDcHDTODDBgDkepOeuk9AE="
+                },
+                "value": {
+                  "denom": "azeta",
+                  "amount": "1000000000000000000000"
+                }
+              }
+            ],
+            "memo": "e3fea0450f9d23ad7b64d41aab882a82a0b71d6b@172.0.1.133:26656",
+            "timeout_height": "0",
+            "extension_options": [],
+            "non_critical_extension_options": []
+          },
+          "auth_info": {
+            "signer_infos": [
+              {
+                "public_key": {
+                  "@type": "/cosmos.crypto.secp256k1.PubKey",
+                  "key": "AmAJR8e3NA7OGpKEIzolKEC+w27XET9KYFW5PmeW/y4r"
+                },
+                "mode_info": {
+                  "single": {
+                    "mode": "SIGN_MODE_DIRECT"
+                  }
+                },
+                "sequence": "0"
+              }
+            ],
+            "fee": {
+              "amount": [],
+              "gas_limit": "200000",
+              "payer": "",
+              "granter": ""
+            },
+            "tip": null
+          },
+          "signatures": [
+            "pcx9op3wpnV+RZgach+esaSsELi3RC7zz/KWwhan/K9XVKEG+fXIFV+LQTWfYmUfi///9b+BYgBFwaVzfVYxyA=="
+          ]
+        },
+        {
+          "body": {
+            "messages": [
+              {
+                "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+                "description": {
+                  "moniker": "validator1-eu-west-1",
+                  "identity": "",
+                  "website": "",
+                  "security_contact": "",
+                  "details": ""
+                },
+                "commission": {
+                  "rate": "0.100000000000000000",
+                  "max_rate": "0.200000000000000000",
+                  "max_change_rate": "0.010000000000000000"
+                },
+                "min_self_delegation": "1",
+                "delegator_address": "zeta1dxyzsket66vt886ap0gnzlnu5pv0y99v086wnz",
+                "validator_address": "zetavaloper1dxyzsket66vt886ap0gnzlnu5pv0y99vt8zj75",
+                "pubkey": {
+                  "@type": "/cosmos.crypto.ed25519.PubKey",
+                  "key": "lexHg6OlvuaLnn2DuCt16o1ZMHGNOG5FmpavdLaANV4="
+                },
+                "value": {
+                  "denom": "azeta",
+                  "amount": "1000000000000000000000"
+                }
+              }
+            ],
+            "memo": "fc5316e6ada821627224a5efa2abb9d9a9c6c8f4@172.28.139.107:26656",
```

Comment:
This is a private IP. Hopefully they've update the public IP properly before TSS signing